cmd/uhc/urls/url_expander.go

```diff
@@ -0,0 +1,56 @@
+package urls
+
+import (
+	"fmt"
+)
+
+// Expand returns full URI to UHC resources based on an alias. An alias
+// allows for shortcuts on the CLI, such as replace "accts" with the
+// full URI of the resource. Lists of resources require just the alias as
+// a parameter, while getting/posting individual resources requires the additional
+// ID of the resource.
+func Expand(argv []string) (string, error) {
+	if len(argv) < 1 || len(argv) > 2 {
+		return "", fmt.Errorf("Invalid parameters. Expected 1 (for Lists) or 2 (for a specific resource) but got %d", len(argv))
```

Comment:
This is sort of redundant with the previous check in the Cmd, but maybe we want it both places.

Reply:
I removed the validation from each of the get/post/patch/delete subcommands in favor of this.